frontend/apps/desktop/src/editor/embed.tsx

```diff
@@ -83,9 +83,11 @@ function Embed({
   const navigateReplace = useNavigate('replace')
   const route = useNavRoute()
 
-  let [docId, version, blockId] = getIdsfromUrl(element.url)
+  let [docId, version, blockId] = getIdsfromUrl((element as EmbedType).url)
```

Comment:
this should be done in one place , probably the prop types. if we have to use `as`, it shouldn't be duplicated if we can avoid it

Reply:
happy to remove duplicates and make it better!. I guess there's an error with the renderElement original function, but not sure